src/execution_plan/ops/op_create.c

```diff
@@ -10,12 +10,13 @@
 #include <assert.h>
 
 // Resolve the properties specified in the query into constant values.
-PendingProperties *_ConvertPropertyMap(Record r, const PropertyMap *map) {
+PendingProperties *_ConvertPropertyMap(GraphContext *gc, Record r, const PropertyMap *map) {
 	PendingProperties *converted = rm_malloc(sizeof(PendingProperties));
 	converted->property_count = map->property_count;
-	converted->keys = map->keys;
+	converted->attr_keys = rm_malloc(sizeof(Attribute_ID) * map->property_count);
```

Comment:
I believe this was my miss understanding, I wanted to reduce the number of calls to `GraphContext_FindOrAddAttribute` suppose we've got one `PropertyMap` object,
`MATCH (n) CREATE (m {n.v + 1})` 
and there are 1M `n` nodes in our graph, as a result `GraphContext_FindOrAddAttribute` will be called 1M times,
I think we should extend `PropertyMap` to include Attribute_ID, this way we'll only need to call `GraphContext_FindOrAddAttribute` once per attribute.
reducing the example above from 1M to just once.